rust-runtime/aws-smithy-http-server/src/routing/mod.rs

```diff
@@ -120,18 +116,6 @@ where
         IntoMakeService::new(self)
     }
 
-    /// Convert this router into a [`MakeService`], that is a [`Service`] whose
-    /// response is another service, and provides a [`ConnectInfo`] object to service handlers.
-    ///
-    /// This is useful when running your application with hyper's
-    /// [`Server`].
-    ///
-    /// [`Server`]: hyper::server::Server
-    /// [`MakeService`]: tower::make::MakeService
-    pub fn into_make_service_with_connect_info<C>(self) -> IntoMakeServiceWithConnectInfo<Self, C> {
```

Comment:
`ConnectInfo` uses the `FromParts` trait which is not functional in the older service builder.